app/uk/gov/hmrc/gform/sharedmodel/formtemplate/Page.scala

```diff
@@ -36,7 +36,7 @@ case class Page(
   continueIf: Option[ContinueIf],
   instruction: Option[Instruction],
   presentationHint: Option[PresentationHint],
-  dataRetrieve: Option[DataRetrieve],
+  dataRetrieve: List[DataRetrieve],
```

Comment:
Lets represent this as `Option[NonEmptyList[DataRetrieve]]` And lets get rid of `withDefaultMissingField`